### ISSUE TITLE
Show tooltips on focus

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet#8111]
**Is your feature request related to a problem? Please describe.**

Tooltips are toggled visible if the element that is bound to it is hovered, but not on focus.

The current behavior doesn't allow keyboard-only users to access tooltips. To confirm this behavior:

1. Go to `debug/map/tooltip.html`
2. Focus any marker that does not have its tooltip initially visible

**Describe the solution you'd like**

I suppose:

- trigger `openTooltip` when the bound element is _focused_ (in concjunction to hover)
- `closeTooltip` if the user moves away from the element

**Additional context**

- https://www.w3.org/TR/wai-aria-practices/#tooltip (emphasis mine):

  > A tooltip is a popup that displays information related to an element _**when the element receives keyboard focus** or the mouse hovers over it_.
- Similar problem to: https://github.com/Leaflet/Leaflet/issues/8078, for keyboard users.
